lib/morandi/image_processor.rb

```diff
@@ -71,8 +71,9 @@ def write_to_png(fn, orientation=:any)
     pb.save(fn, 'png')
   end
 
-  def write_to_jpeg(fn, quality = 97)
-    @pb.save(fn, 'jpeg', :quality => quality)
+  def write_to_jpeg(fn, quality = nil)
+    quality ||= options.fetch(:quality, 80)
```

Comment:
Support for setting quality in options hash has been silently added, while should be mentioned at least in the commit message